hashmail_server.go

```diff
@@ -198,12 +205,27 @@ func newStream(id streamID, limiter *rate.Limiter,
 	writeReadPipe, writeWritePipe := io.Pipe()
 	readReadPipe, readWritePipe := io.Pipe()
 
+	staleTimer := time.AfterFunc(staleTimeout, func() {
```

Comment:
Should we allow to disable this feature, for example by setting a value of `-1`? Just in case we notice it's breaking something and we need a quick fix.

Reply:
I think we should either move the timer into the `streamStatus` and make it less generic or add the `staleTimer.Stop()` to the `tearDown` func.

Reply:
> Should we allow to disable this feature, for example by setting a value of `-1`? Just in case we notice it's breaking something and we need a quick fix.

Great idea! Added 👍